### DISCUSSION TITLE
Should I stop trying to install Open Neptune?

### DISCUSSION THREAD

**Discussion by paulofaria1** [OpenNeptune3D/OpenNept4une#228] [Q&A]
Hello guys.
It´s been a ride, trying to install Open Neptune. After 3 days, the last obstacle that I definitely can´t overcome is the Ethernet connection lost. After the "reboot" part, on the wiki, the ethernet never came back. Tried everything. Changed cables, changed router ports, etc. Even called the router support. So, I´m stucked. Oh, and flashed the EMMC again. I´m on a MAC without physical USB port (I use adapters), so the putty solution for usb doesn't work for me. Any ideas?

Thanks :)

**Comment by skipster74** [comment]
Sorry you're having problems. You're not alone if that helps. 

Like you I flash the emmc and then reboot. I get ethernet activity for a few minutes then cuts off. During that time I can't connect via ssh so I feel like I'm hitting a wall.  I'm also on a Mac.

**Reply by paulofaria1** [reply]
I'm still looking for an answer. Wrote to Elegoo, but they told me to install the firmware. Well, how can I? Don't have connection.
Let's change some tips here if we find some useful ones.

**Comment by skipster74** [comment]
That was my next quest: is there a full image that’s already updated? I read the elegoo discord is a go-to. I’m going to flash my Emmc again before trying that.

**Comment by skipster74** [comment]
So after a second flashing I did get it to work…kinda sorta almost. 

The fan mounted on the bottom cover the sits below the motherboard isn’t turning on and as a result, the unit shuts -own due to overheating.  ☹️

So next up I’ll try reseating the fan connection and failing that, replace it.

**Reply by paulofaria1** [reply]
Good for you. I tried to flash the emmc a dozen times, without success.
